src/components/toast/global_toast_list.tsx

```diff
@@ -37,6 +46,8 @@ export interface EuiGlobalToastListProps extends CommonProps {
   toasts: Toast[];
   dismissToast: (this: EuiGlobalToastList, toast: Toast) => void;
   toastLifeTimeMs: number;
+  //side prop determines which side(right or left) will the toast appear
```

Comment:
In order for the props comment to be properly picked up by IDE's and our props table, it must be formatted like so:
```suggestion
  /**
   * Determines which side of the browser window the toasts should appear
   */
```